auto_nag/scripts/component.py

```diff
@@ -49,22 +50,13 @@ def get_bz_params(self, date):
             'chfield': '[Bug creation]',
             'chfieldfrom': start_date,
             'chfieldto': end_date,
+            # Ignore bugs for which somebody has ever modified the product or the component.
+            'n1': 1, 'f1': 'product', 'o1': 'changedafter', 'v1': datetime(1970, 1, 1),
```

Comment:
nit: you can remove datetime and just use '1970-01-01'

Reply:
Done with de2ed31.